lxd/resource_lxd_container.go

```diff
@@ -309,6 +309,9 @@ func resourceLxdContainerExists(d *schema.ResourceData, meta interface{}) (exist
 	exists = false
 
 	ct, err := client.ContainerState(name)
+	if err != nil && err.Error() == "not found" {
+		err = nil
```

Comment:
Although, on second thought this is in the Exists func making it subtly different to the Google example.